README.md

```diff
@@ -22,12 +22,20 @@ Then, at the top of your `Rakefile`, add:
 require 'voxpupuli-release'
 ```
 
-To cut a new release of your module, ensure the `metadata.json` reflects the proper version. Also ensure that the `CHANGELOG.md` has a note about the release and that it actually is named Release or release, some old modules refer to it as Version, which won't work. Lastly check that no tag exists with that version number (format `v#.#.#`), and then run:
+To cut a new release of your module, ensure the `metadata.json` reflects the expected new version of the module, that no tag exists with that version number (format `v#.#.#`), and then update the module `CHANGELOG.md` and `REFERENCE.md` by running:
+
+```plain
+bundle exec rake release:prepare
+```
+
+Commit these changes (likely in a new branch, open a Pull-Request and wait for it to be reviewed and merged).  When ready to ship the new release, ensure you are on the main branch, it is up-to-date, and run:
```

Comment:
Is it possible for the task to create a new branch and create the commit with a standardised commit message?  I'd like to see `Release x.x.x` used consistently.

Reply:
/me feels like it is too much :smile: 

Would we can just adjust the message and suggest things:

```
Please review these changes and commit them to a new branch:

    git checkout -b release-x.x.x
    git commit -m "Release x.x.x"

Then open a Pull-Request and wait for it to be reviewed and merged).
```

Reply:
I had this as local changes in my working directory, so I added a commit with it.